docs/pages/reference/mesheryctl-commands.md

```diff
@@ -1,401 +1,402 @@
+          <td></td>
+          <td>{{ subcommand.description }}</td>
+        </tr>
+      {% endfor %}
+      {% for subcommand_hash in site.data.mesheryctlcommands.lifecycle.system-context.view.command %}{% assign subcommand = subcommand_hash[1] %}
+        <tr>
+          <td rowspan=3><a href="{{ site.baseurl }}/reference/mesheryctl/system/context/view">{{ subcommand.name }}</a></td>
+          <td></td>
+          <td>{{ subcommand.description }}</td>
+        </tr>
+      {% endfor %}
+      {% for flag_hash in site.data.mesheryctlcommands.lifecycle.system-context.view.flags %}{% assign flag = flag_hash[1] %}
+        <tr>
+          <td>{{ flag.flag }}</td>
+          <td>{{ flag.description }}</td>
+        </tr>
+      {% endfor %}
+      {% for subcommand_hash in site.data.mesheryctlcommands.lifecycle.system-context.switch.command %}{% assign subcommand = subcommand_hash[1] %}
+        <tr>
+          <td rowspan=4><a href="{{ site.baseurl }}/reference/mesheryctl/system/context/switch">{{ subcommand.name }}</a></td>
+          <td></td>
+          <td>{{ subcommand.description }}</td>
+        </tr>
+      {% endfor %}
+      {% for flag_hash in site.data.mesheryctlcommands.lifecycle.system-context.switch.flags %}{% assign flag = flag_hash[1] %}
+        <tr>
+          <td>{{ flag.flag }}</td>
+          <td>{{ flag.description }}</td>
+        </tr>
+      {% endfor %}
+  {% endfor %}
+</thead>
+</table>
+
+## Service Mesh Performance Management
+
+<table>
+<thead>
+  <tr>
+    <th>Main Command</th>
+    <th>Flag</th>
+    <th>Function</th>
+  </tr>
+  {% for command_hash in site.data.mesheryctlcommands.performance.commands %}{% assign command = command_hash[1] %}
+    <tr>
+      <td rowspan=11><a href="{{ site.baseurl }}/reference/mesheryctl/perf">{{ command.name }}</a></td>
+      <td></td>
+      <td>{{ command.description }}</td>
+    </tr>
+    {% for flag_hash in site.data.mesheryctlcommands.performance.flags %}{% assign flag = flag_hash[1] %}
+      <tr>
+        <td>{{ flag.name }}</td>
+        <td>{{ flag.description }}</td>
+      </tr>
+    {% endfor %}
+  {% endfor %}
+</thead>
+</table>
+
+
+## Service Mesh Lifecycle and Configuration Management
+
+<table>
+<thead>
+  <tr>
+    <th>Main Command</th>
+    <th>Command</th>
+    <th>Flag</th>
+    <th>Function</th>
+  </tr>
+  {% for command_hash in site.data.mesheryctlcommands.meshes.commands %}{% assign command = command_hash[1] %}
+    <tr>
+      <td rowspan=10><a href="{{ site.baseurl }}/reference/mesheryctl/mesh">{{ command.name }}</a></td>
+      <td></td>
+      <td></td>
+      <td>{{ command.description }}</td>
+    </tr>
+  {% endfor %}
+    {% for subcommand_hash in site.data.mesheryctlcommands.meshes.init.commands %}{% assign subcommand = subcommand_hash[1] %}
+      <tr>
+        <td rowspan=4>{{ subcommand.name }}</td>
+        <td></td>
+        <td></td>

```

Comment:
@leecalcote : I delete folder i.e, overedited & again download the initial folder, while pushing it says `branch has updated`, so I  just added `<td></td>` to reinitialize & then, commited the changes with `All-in-one changes`.
I think, this falls in my mistake!